app/views/components/TaskItem.scala.html

```diff
@@ -25,7 +25,7 @@
 
 <li class="app-task-list__item" id="task-list__item--@{task.link.text}">
 
-    <span class="app-task-link">
+    <div class="app-task-link govuk-!-width-two-thirds">
```

Comment:
Interested why this was made a div, guessing we wanted a `block` element rather than an `inline` or `inline-block` element for the css box model? A `div` inside of an `li`will be flagged as invalid HTML. We can probably achieve this using custom css if need be.

Reply:
It was indeed because I did not want it inline, will message you.

Reply:
it was decided we will fix this as a defect. So that we can unblock this journey.